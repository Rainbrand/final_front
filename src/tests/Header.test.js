import React from "react";
import {shallow} from "enzyme";
import ChatHeader from "../components/ChatHeader";
const Chance = require('chance');
const chance = new Chance();

describe("ChatHeader component should:", () => {
    let component

    beforeAll(() => {
        component = shallow(<ChatHeader className="header"
                                        lastSeen={chance.timestamp()} users={currentChat.users}/>)
        console.log(component.debug())
    })

    test("render ChatHeader component", () => {
        expect(component).toMatchSnapshot()
    })

    test('have props', () => {
        expect(component.props()).not.toBeUndefined()
    })

})
