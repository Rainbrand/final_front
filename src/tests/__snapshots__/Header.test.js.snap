// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatHeader component should: render ChatHeader component 1`] = `ShallowWrapper {}`;
